### DISCUSSION TITLE
Getting Error - 'React/RCTFabricSurfaceHostingProxyRootView.h' file not found on React Native 0.74.5

### DISCUSSION THREAD

**Discussion by abhayagrawal-fareye** [reactwg/react-native-new-architecture#233] [Q&A]
I get this error after upgrading to RN 0.74.5 and enabling the new architecture.

Read through this https://github.com/facebook/react-native/commit/33e0521788484eaba20beeeaabba2496854583a7#diff-e358c37a21b2125a2756aa69329f878b208537770e971dac49651652d67ef218R139 and 
this file is present in my node-modules  

Xcode 16

I have realm as a static framework and is added by SPM. Realm version - 10.54.0
Also, I am using Firebase which might have caused this issue.

These files are present in Podfile

`pod 'Firebase/Analytics'
pod 'Firebase/Crashlytics'
pod 'FirebaseRemoteConfig'`

`# Apply modular headers to their key dependencies
pod 'FirebaseCore', :modular_headers => true
pod 'FirebaseInstallations', :modular_headers => true
pod 'GoogleUtilities', :modular_headers => true
pod 'GoogleDataTransport', :modular_headers => true
pod 'nanopb', :modular_headers => true
pod 'FirebaseABTesting', :modular_headers => true
pod 'FirebaseCoreExtension', :modular_headers => true
`

**Comment by cortinico** [comment]
Please open an issue here:
https://github.com/facebook/react-native/issues/new/choose

and make sure to provide a reproducer.

**Comment by billyprg** [comment]
**Reason:** Several Fabric-related headers have been moved to new locations. 
**Solution:**  Replace these in AppDelegate
 from:
 ```
 #import <React/RCTFabricSurfaceHostingProxyRootView.h>
#import <React/RCTSurfacePresenter.h>
#import <React/RCTSurfacePresenterBridgeAdapter.h>
```

 To:  
```
 #import <React-RCTFabric/RCTFabricSurfaceHostingProxyRootView.h>
#import <React-RCTFabric/RCTSurfacePresenter.h>
#import <React-RCTFabric/RCTSurfacePresenterBridgeAdapter.h>

```
